src/components/PeoplePicker/PeoplePicker.tsx

```diff
@@ -507,7 +507,7 @@ export class PeoplePicker extends React.Component<IPeoplePickerProps, IPeoplePic
                   <button className='ms-PeoplePicker-resultAction' onClick={ () => {
                       this._removeSelectedPersona( selectedPersonas.indexOf(child) );
                   }}>
-                    <i className='ms-Icon ms-Icon--Cancel'></i>
+                    <i className='ms-Icon ms-Icon--Clear'></i>
```

Comment:
Why change it to Clear from Cancel? @Jahnp Isn't ms-Icon--x mapping to ms-Icon--Cancel?